customer/interfaces/graphql/schema.graphql

```diff
@@ -0,0 +1,43 @@
+type Commerce_Customer_StatusResult {
+    isLoggedIn: Boolean!
+    userID: String!
+}
+
+type Commerce_CustomerResult {
+    id: String!
+    personalData: Commerce_Customer_PersonData!
+    addresses: [Commerce_Customer_Address!]
+    defaultShippingAddress: Commerce_Customer_Address!
+    defaultBillingAddress: Commerce_Customer_Address!
+}
+
+type Commerce_Customer_PersonData {
+    gender:     String!
+    firstName:  String!
+    lastName:   String!
+    middleName: String!
+    mainEmail:  String!
+    prefix:      String!
+    birthday:    Time!
+    nationality: String!
+}
+
+type Commerce_Customer_Address {
+    regionCode:             String!
+    countryCode:            String!
+    company:                String!
+    street:                 String!
+    streetNr:               String!
+    additionalAddressLines: [String!]
+    telephone:              String!
+    postCode:               String!
+    city:                   String!
+    firstname:              String!
+    lastname:               String!
+    email:                  String!
+}
+
+extend type Query {
```

Comment:
maybe worth to add some description in the schema. so we can benefit from the auto generated docs in the graphql playground.
http://spec.graphql.org/June2018/#sec-Descriptions